locales/en.json

```diff
@@ -1,7 +1,29 @@
 {
+  "ErrorIndividualPackagesUnsupported": "Error: In manifest mode, `vcpkg install` does not support individual package arguments.\nTo install additional packages, edit vcpkg.json and then run `vcpkg install` without any package arguments.",
+  "ErrorInvalidClassicModeOption": "Error: The option {value} is not supported in classic mode and no manifest was found.",
+  "ErrorInvalidManifestModeOption": "Error: The option {value} is not supported in manifest mode.",
+  "ErrorNoVSInstance": "Error: in triplet {triplet}: Unable to find a valid Visual Studio instance",
+  "ErrorNoVSInstanceAt": "     at \"{path}\"",
```

Comment:
Should these extra spaces be part of the localized text? It seems that should be an unlocalized (formatting) part.

Reply:
I think adding unlocalized formatting support to the localization stuff would be a lot for not a lot of benefit.